### PR TITLE
[Excel] (Custom Functions) Update custom-functions-json.md

### PR DESCRIPTION
**Pull request by keyur32**
Internal bug: 
Bug 3287052: Doc Bug: requiresAddress missing in metadata documentation

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[ce1ecee](https://github.com/OfficeDev/office-js-docs-pr/pull/888/commits/ce1eceea8e6ab0751712e19743571c390c418fbc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/metadata-requiresAddress/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C10%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904102255246182-888%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=mC2aUu9DZk8vIrFNrNIypcGPa%2FIpyeGoe3coVSbh%2Bh8%3D&st=2019-04-10T22%3A51%3A49Z&se=2019-05-11T22%3A56%3A49Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-json.md [hunk 1]
```diff
@@ -131,6 +131,7 @@ The `options` object enables you to customize some aspects of how and when Excel
...
+|  `requiresAddress`  | boolean | No <br/><br/>Default value is `false`. | <br /><br /> If `true`, the function will be passed the address of the cell that invoked your custom function. In this option, Excel will call the JavaScript function with the last parameter being an 'invocatationContext' paramter. In the body of the function, if `requiresAddress` is set, you can access `context.address` member. A function cannot be both streaming and require an address. For more information, see [Determine which cell invoked your custom function](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-overview#determine-which-cell-invoked-your-custom-function). |
```
Comment (Reezaali) [inline docs/excel/custom-functions-json.md]: @keyur32 can you change the text as follows if it is correct please: 

If true, your custom function can access the address of the cell that invoked your custom function. To get the address of the cell that invoked your custom function, use context.address in your custom function. For more information, see [Determine which cell invoked your custom function](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-overview#determine-which-cell-invoked-your-custom-function). Custom functions cannot be set as both streaming and requiresAddress. When using this option, the 'invocatationContext' parameter must be the last parameter passed in options.

Comment (mscharlock) [reply docs/excel/custom-functions-json.md]:
```suggestion
|  `requiresAddress`  | boolean | No <br/><br/>Default value is `false`. | <br /><br /> If true, your custom function can access the address of the cell that invoked your custom function. To get the address of the cell that invoked your custom function, use context.address in your custom function. For more information, see [Determine which cell invoked your custom function](/office/dev/add-ins/excel/custom-functions-overview#determine-which-cell-invoked-your-custom-function). Custom functions cannot be set as both streaming and requiresAddress. When using this option, the 'invocationContext' parameter must be the last parameter passed in options. |
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[409e381](https://github.com/OfficeDev/office-js-docs-pr/pull/888/commits/409e381b3f1f6b241913f8068bb5343ba6eaeb9a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/metadata-requiresAddress/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C15%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904152247016157-888%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=LHs%2FbQm2of583VVHawygpYKh7eTb8DNco145bu3Ql%2FE%3D&st=2019-04-15T22%3A43%3A33Z&se=2019-05-16T22%3A48%3A33Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**